### PR TITLE
feat(email template): created decline registration template

### PR DESCRIPTION
**Pull request by nidhigarg-bmw**
## Description

Created Decline Registration template

## Why

When user decline the request then this template will be sent

## Issue

n/a

## Checklist

Please delete options that are not relevant.

- [X] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [X] I have performed [IP checks](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-04#checking-libraries-using-the-eclipse-dash-license-tool) for added or updated 3rd party libraries
- [X] I have performed a self-review of my own code
- [X] I have successfully tested my changes locally

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -38,7 +38,9 @@ New features, fixed bugs, known defects and other noteworthy changes to each rel
   * updated and added technical user role description & user role names
 * Others
   * added email value validation for invitation, network registration and user invitation to enable the user input data validation for valid email
-
+* Email templates
+  * released new email template 'decline registration'
```
Comment (jjeroch) [inline CHANGELOG.md:42]: @nidhigarg-bmw correction needed. You have by mistake added the changeLog item in the closed section for 1.7.0 release. Please delete this again. No changeLog update in the backend repo

**Review by jjeroch — CHANGES_REQUESTED** [review]

**Review by Phil91 — APPROVED** [review]

**Comment by jjeroch** [comment]
@nidhigarg-bmw please have a look at the sonar fails. Ideally also sync with Mano, he did a number of the other email templates and might have useful hints to fix the issue (if needed)

**Comment by nidhigarg-bmw** [comment]
> @nidhigarg-bmw please have a look at the sonar fails. Ideally also sync with Mano, he did a number of the other email templates and might have useful hints to fix the issue (if needed)

@jjeroch I already discussed with Mano, he told that they are known issues in all the templates. We can ignore these for now.

**Comment by jjeroch** [comment]
> > @nidhigarg-bmw please have a look at the sonar fails. Ideally also sync with Mano, he did a number of the other email templates and might have useful hints to fix the issue (if needed)
> 
> @jjeroch I already discussed with Mano, he told that they are known issues in all the templates. We can ignore these for now.

@manojava-gk  & @nidhigarg-bmw  in sonar it shows the issue as "unresolved" ideally we decide/switch the status to "Won't Fix" or "False Positive"  (if this make sense) - please sync once if we ideally change the status of all those findings instead of leaving is "unresolved" - unresolved has always the message that something still needs to be done 
image

If you have troubles to update the status, please ping @Phil91

**Pull request merged**